src/components/RestaurantDetails/RestaurantDetails.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
```

Comment:
this could be broken down to simpler components and the data could be loaded from config file instead of hardcoding

Reply:
++

> this could be broken down to simpler components and the data could be loaded from config file instead of hardcoding

++